app/models/user.rb

```diff
@@ -12,4 +12,9 @@ class User < ApplicationRecord
 
   has_many :book_presentations, dependent: :destroy
   has_many :books, -> { distinct }, through: :book_presentations
+
+  enum role: {
+    user: "user",
+    admin: "admin"
+  }
 end
```

Comment:
Is this addition demanded by the administrate gem or can the already existing moderator flag be used instead?

Reply:
Ignore this I'll delete